util/moaiutil.lua

```diff
@@ -5,11 +5,73 @@ require ( 'http' )
 -- setup
 --==============================================================
 
-INVOKE_DIR		= MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 1 ])
-MOAI_SDK_HOME	= MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 2 ])
-MOAI_CMD		= arg [ 3 ]
-SCRIPT_DIR		= string.format ( '%sutil/%s/', MOAI_SDK_HOME, MOAI_CMD )
+INVOKE_DIR      = MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 1 ])
+MOAI_SDK_HOME   = MOAIFileSystem.getAbsoluteDirectoryPath ( arg [ 2 ])
+MOAI_CMD        = arg [ 3 ]
+SCRIPT_DIR      = string.format ( '%sutil/%s/', MOAI_SDK_HOME, MOAI_CMD )
+
+function usage()
+  print([[
+    MOAI Utility Toolbelt 
+
+    PREPARE to use the toolbelt:
+        MOAI SDK has a toolbox environment script (env.sh) for use prior to loading the toolbelt:
+            source $MOAI_SDK_HOME/env.sh
+        (Presumably you found it already.)
+    
+    INITIALIZE Project folder:              moaiutil init
+        Run from a clean project folder to initialize a moaiutil-managed set of host projects.
+        Creates a hostconfig.lua file, which you must edit to describe your MOAI project.
+        Example:
+            mkdir newMOAIProject && cd newMOAIProject   # use the toolbelt in your own project workspace
+            moaiutil init 
+            vi hostconfig.lua                           #&etc.
+
+    CREATE and Manage MOAI Projects:        moaiutil host <subcommand> <args>
+        Subcommands:
+            host list - Lists available hosts
+            host init - Creates a template host config file used by subsequent commands
+            host build <hostname> - Creates (if it doesn't exist) in hosts folder, and (re)builds the host named <hostname>. 
+            host run <hostname> - Creates (if it doesn't exists) in hosts folder, (re)builds and runs the host named <hostname>. 
+            host create <hostname> - Creates the host in the hosts folder (removing old host) based on latest config settings.
+        Example:
+            cd newMOAIProject && moaiutil host create ios && moaiutil host create android-studio && moaiutil host create host osx-app && #etc.
+
+    MAKE Developer Documentation:           moaiutil make-cpp-docs
+        Creates compact documentation from the MOAI C/C++ codebase.
+        Example:    
+            cd newMOAIProject && moaiutil make-cpp-docs
```

Comment:
this guy is just a stub atm, would be a good idea to mention moaiutil make-lua-docs though